### DISCUSSION TITLE
I can't find the resampling option

### DISCUSSION THREAD

**Discussion by Alexanvonner** [bangcorrupt/hacktribe#199] [Q&A] [answered]
hello thank you for this great hacktribe
I only have one question I would like to create a kick and do a resampling (button shift + rec for access to resampling) but I no longer see resampling on the right it's only mono / audioIN / 273sec

I thank you in advance for your precious help.

**Comment by bangcorrupt** [comment, marked as answer]
> Audio In

Turn the main encoder.
